### DISCUSSION TITLE
Pimcore toolbox does not appear on content page -> where is it gone??

### DISCUSSION THREAD

**Discussion by marrep** [pimcore/pimcore#9238] [Q&A / Support] [answered]
hi there,

I ve installed the [pimcore toolbox bundle](https://github.com/dachcom-digital/pimcore-toolbox) which is now clearly visible in my composer.json

````
"require": {
  "php": ">=7.2",
  "dachcom-digital/toolbox": "^3.2",
  "pimcore/pimcore": "~6.8.0"
},
````
Neither I find the area bricks in _src/AppBundle/Document/Areabrick_ (...only my custom bricks that I ve created) nor I see them when initiating a regular content page by using 
{{ pimcore_areablock("content")}}

Any hint on where to find and how to implement them in my project?

Cheers!

**Comment by passioneight** [comment, marked as answer]
The bundle you installed is neither created nor maintained by Pimcore.  You should probably create an issue there, to see what's going on. I'm sure the guys at Dachom Digital can help you out.

In any case, I'd also check if the bundle is enabled and installed first: `bin/console pimcore:bundle:list`.

**Reply by marrep** [reply]
Thanks I solved it - Indeed I skipped one part of the documentation which points out to enable the extension after installation in the admin > bundles > extensions by clicking "+" .
